<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RahanSiirto</class>
 <widget class="QDialog" name="RahanSiirto">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>20</pointsize>
   </font>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Tilisiirto</string>
  </property>
  <widget class="QPushButton" name="palaaButton">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>470</y>
     <width>300</width>
     <height>100</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Peruuta</string>
   </property>
  </widget>
  <widget class="QPushButton" name="jatkaButton">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>470</y>
     <width>300</width>
     <height>100</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Jatka</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>40</y>
     <width>291</width>
     <height>70</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Siirrettävä rahamäärä:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="SiirtoEdit">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>40</y>
     <width>300</width>
     <height>70</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>00,00</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>250</y>
     <width>300</width>
     <height>70</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>FI</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>250</y>
     <width>161</width>
     <height>70</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Tilinumero:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>150</y>
     <width>271</width>
     <height>70</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Vastaanottajan nimi:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="NimiEdit">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>140</y>
     <width>300</width>
     <height>70</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="viestiLabel">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>360</y>
     <width>101</width>
     <height>70</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Viesti:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="viestiEdit">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>360</y>
     <width>300</width>
     <height>70</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>40</y>
     <width>80</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Num</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>140</y>
     <width>80</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Keys</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>250</y>
     <width>80</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Num</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>360</y>
     <width>80</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Keys</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
